modules/olm-about-catalogs.adoc

```diff
@@ -22,3 +22,10 @@ As a cluster administrator, you can create your own custom index image, either b
 ====
 When creating custom catalog images, previous versions of {product-title} 4 required using the `oc adm catalog build` command, which was deprecated for several releases and is now removed. With the availability of Red Hat-provided index images starting in {product-title} 4.6, catalog builders must use the `opm index` command to manage index images.
 ====
+
+[IMPORTANT]
+====
+Kubernetes periodically deprecates certain APIs that are later removed in subsequent releases. As a result, Operators will be unable to use any removed APIs starting with the version of {product-title} that uses whichever Kubernetes version removed the API.
+
+If you are working with custom catalogs, see xref:../../operators/operator_sdk/osdk-working-bundle-images#osdk-control-compat_osdk-working-bundle-images[Controlling Operator compatibility with {product-title} versions] for more details about how Operator authors can update their projects to help avoid workload issues and prevent incompatible upgrades.
+====
```

Comment:
Warning: low coffee intake at the time of this review.  Ignore what you don't like.

* Maybe remove "later" from "later removed."  Later and subsequent in close proximity caught my uncaffeinated eye.
* Maybe s/will be/are/ and s/any//
* Maybe "...that uses the Kubernetes version that removed the API."
* Maybe "If your Operator works with custom catalogs,..." ("are working" can "work" either way)

Reply:
✅
✅
✅
✅